bot/actions/slack_incident_listener.rb

```diff
@@ -4,7 +4,12 @@
 
     begin
       if action[:payload][:view][:app_id] == ENV['SLACK_APP_ID']
-        SlackIncidentActions.new.open_incident(action)
+        if action[:payload][:view][:callback_id] == 'incident-update-modal-identifier'
+          channel_id = Rails.cache.read('channel')
```

Comment:
I think we established we could make this cache key unique per channel? Not needed for now tho!

Reply:
It was originally like [this](https://github.com/DFE-Digital/slack-incident-bot/pull/11/files#diff-41d8fdcddd9bfc947ed7decf57f8a3f04acbe2c48f0f07809b40f6971a09adaeR13)... but I couldn't get it to work on this PR.. so went with the above 🤷‍♂️